BridgeSDK/Networking/SBBNetworkManager.m

```diff
@@ -692,6 +683,16 @@ - (void)handleError:(NSError*)error task:(NSURLSessionDataTask*) task retryObjec
     }
 }
 
+- (void)handleHTTPError:(NSError *)error task:(NSURLSessionDataTask *)task retryObject:(APCNetworkRetryObject *)retryObject
+{
+    //TODO: Add retry for Server maintenance
```

Comment:
Actually no, I believe this is a comment that was in the original networking code provided to us by YML and now that you bring it up, I'm not sure it's relevant anymore--this won't get called for background NSURLSession tasks, and likely isn't really appropriate when the UI might be waiting on a response.